test/e2e/csv_e2e_test.go

```diff
@@ -4432,17 +4432,18 @@ func fetchCSV(c versioned.Interface, name, namespace string, checker csvConditio
 	var fetchedCSV *operatorsv1alpha1.ClusterServiceVersion
 	var err error
 
+	ctx.Ctx().Logf("waiting for CSV %s/%s to reach condition", namespace, name)
```

Comment:
If a function like `fetchCSV` is going to validate conditions, at least log something to make it less surprising.